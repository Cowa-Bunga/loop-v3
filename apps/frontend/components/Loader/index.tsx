// TODO:
const Loader = () => <>...</>

export default Loader
