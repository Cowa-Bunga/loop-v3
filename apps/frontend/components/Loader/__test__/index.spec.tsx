import React from 'react'
import { render } from '@testing-library/react'
import Loader from '../index'

describe('Loader', () => {
  it('should render successfully', () => {
    const { baseElement } = render(<Loader />)
    expect(baseElement).toBeTruthy()
    expect(baseElement).toMatchSnapshot()
  })
})
