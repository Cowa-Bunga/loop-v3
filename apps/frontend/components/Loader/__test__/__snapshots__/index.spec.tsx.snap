// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader should render successfully 1`] = `
<body>
  <div>
    ...
  </div>
</body>
`;
